app/forms/recovery_code_form.rb

```diff
@@ -25,9 +25,9 @@ def valid_recovery_code?
     recovery_code_generator.verify(code)
   end
 
-  def result
+  def extra_analytics_attributes
     {
-      success: success,
+      method: 'recovery code',
```

Comment:
I default to constantizing most strings when their value is important for categorization purposes. 

Is it important for this string to have a consistent value for analytics stuff?